common/versioning/version.properties

```diff
@@ -1,3 +1,4 @@
-#Thu Aug 02 12:03:16 PDT 2018
-versionName=3.0.5-RC1
-versionCode=1
+#Mon Nov 02 10:02:30 PST 2020
+versionName=0.0.26
+versionCode=1
+latestPatchVersion=26

```

Comment:
I'm not clear on what is happening here?

Reply:
This is the version file that we've been using to actually version things.  The versionName shouldn't be set to this, but I'll change before merging.  I'll write up some documentation to go along with this PR.

Reply:
I know the file....I don't understand what these `^M` symbols are meant to do

Reply:
and why the version is down to from `3.0.5` to `0.0.26`